docs/esp8266/tutorial/intro.rst

```diff
@@ -109,10 +109,15 @@ PC.  You may also need to reduce the baudrate if you get errors when flashing
 that you have.
 
 For some boards with a particular FlashROM configuration (e.g. some variants of
-a NodeMCU board) you may need to use the following command to deploy
-the firmware (note the ``-fm dio`` option)::
+a NodeMCU board) you may need to manually set a compatible
+`SPI Flash Mode <https://github.com/espressif/esptool/wiki/SPI-Flash-Modes>`_:
 
-    esptool.py --port /dev/ttyUSB0 --baud 460800 write_flash --flash_size=detect -fm dio 0 esp8266-20170108-v1.8.7.bin
+    "In general, choose the fastest option for flash_mode that works with your
+    device. Not all devices support all modes."
```

Comment:
What's the intention of the formatting of this section?  It might look better just as normal text in a normal paragraph (and as part of the previous paragraph).

Reply:
That paragraph is a direct quote from the esptool wiki which is linked immediately before that. That's why I put it into quotes and indented the section as a RST quote.